presto-docs/src/main/sphinx/statistics/statistics.rst

```diff
@@ -0,0 +1,155 @@
+================
+Table statistics
+================
+
+Presto supports statistics based optimizations for queries. For query to be subject for such optimization
+statistics information must be available for tables used by query.
+
+Table statistics are provided to query planner by connectors.
+Currently the only connector supporting the mechanism is Hive connector.
+
+Table layouts
+-------------
+
+Statistics are exposed to query planner on per table layout basis. Table layout represents a subset of table's data with attached
+data organization properties (like sort order, bucketing etc.).
+
+How many table layouts table has and details of table layout implementation are specific to connector.
+Taking Hive connector as an example:
+
+* not partitioned tables have just one table layout - representing all data in a table
+* partitioned tables have a family of table layouts. Each subset of partitions to be scanned represent one table layout.
+  Presto will try to pick up smallest possible layout based on filtering predicates from query.
+
+Available statistics
+--------------------
+
+Currently following statics are available in Presto:
+ * **rows count** - total number of rows for table layout
+ * Per each column:
```

Comment:
How about minValue / maxValue for a given column? Reading https://cwiki.apache.org/confluence/display/Hive/Column+Statistics+in+Hive makes we believe they should be available.